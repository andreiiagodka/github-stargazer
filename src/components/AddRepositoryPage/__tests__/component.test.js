import { shallow } from 'enzyme'

import AddRepositoryPageComponent from '..'

describe('<AddRepositoryPageComponent />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<AddRepositoryPageComponent />)

    expect(wrapper).toMatchSnapshot()
  })
})