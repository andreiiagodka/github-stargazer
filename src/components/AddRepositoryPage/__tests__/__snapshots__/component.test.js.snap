// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddRepositoryPageComponent /> renders correctly 1`] = `
<LayoutComponent
  body={<ContentComponent />}
  header={<HeaderComponent />}
/>
`;
